xdsl/frontend/onnx/context.py

```diff
@@ -0,0 +1,76 @@
+from onnx import GraphProto, NodeProto, ValueInfoProto
+
+from xdsl.builder import ImplicitBuilder
+from xdsl.dialects import func, onnx
+from xdsl.dialects.builtin import ModuleOp
+from xdsl.frontend.onnx.type import get_type
+from xdsl.ir import Attribute, SSAValue
+from xdsl.irdl import IRDLOperation
+
+OP_BY_OP_TYPE: dict[str, type[IRDLOperation]] = {
+    "Add": onnx.Add,
+    "Sub": onnx.Sub,
+}
+"Dictionary that associates the name of the operations with the respective operation in ONNX dialect."
```

Comment:
I'd recommend skipping `Dictionary` here, and `class` below. The client already knows that this is a dictionary and that `Ctx` is a class. Can you please rephrase the comments to just say what the thing does, and not what kind of thing it is?

Also, could you please use triple quotes for doc comments?